### ISSUE TITLE
apply rules to region

### ISSUE THREAD

**Issue by fkgruber** [davidshepherd7/electric-operator#20]
Hi
It would be a nice feature to be able to select a region of existing code and apply the electric-operator rules to it. Essentially make existing code look better.

thanks
F

**Comment by davidshepherd7** [comment]
Yeah this might be nice, I'll have a think about how it can be implemented. If I can do it without having to change how existing rules work then I'll do it.

It's also worth noting that there are plenty of tools for "offline" beautification of code. For example python has [autopep8](https://pypi.python.org/pypi/autopep8/) which will apply the same spacing rules as electric-operator's defaults. These tools are usually more powerful than anything I could or would add.
